src/main/java/com/javadiscord/javabot/commands/configuation/config/Config.java

```diff
@@ -37,8 +37,10 @@ public Config() {
     @Override
     public ReplyAction handle(SlashCommandEvent event) {
 
-        try { return super.handle(event);
-        } catch (Exception e) { return Responses.error(event, "```" + e.getMessage() + "```"); }
+        return Responses.error(event, "This command is currently not available.");
```

Comment:
I would prefer a `Responses.warning`, since an error hints that something serious went wrong, whereas warning is just telling them that the user is doing something wrong.